RELEASING.md

```diff
@@ -1,9 +1,29 @@
 # Releasing
 
-1. Update the `version` in [gradle.properties](gradle.properties) to a non-`SNAPSHOT`.
-2. Update [CHANGELOG.md](CHANGELOG.md)
-  - Add the new version section and move the *unreleased changes* into it.
-  - Update the links at the end of the page.
+## Pre-release
+
+Before each release we will publish a Snapshot version of it and use it to create a pre release on [Github Release page](https://github.com/adevinta/spark-android/releases)
+
+During a period of ~2 weeks consumers will be able to test it for bugs that can be fixed before it’s considered stable.
+
+Using Github pre-release feature will allow Spark users to create a hook to notice that a new release has been made and test against it.
+For example they could hook to it to make a bot post this changelog in their monitoring slack channel or trigger a CI build to certify that this new release doesn’t break their build.
+
+1. First create a draft release that by creating a `*.*.*-SNAPSHOT` version tag.
+2. Use this new tag to use the change log generation tool from GitHub.
+3. Remove logs from Dependabot except if they mention big version upgrade for libraries used by our consumers (like Compose or Kotlin version upgrade).
```

Comment:
```suggestion
@dependabot
3. Remove logs from @dependabot except if they mention big version upgrades for libraries used by our consumers (like Compose or Kotlin).
```

Reply:
Done in https://github.com/adevinta/spark-android/commit/6730295a22b486d089a1d554ba9e1e2add81719b